pkg/manifest/os.go

```diff
@@ -791,6 +793,20 @@ func (p *OS) serialize() osbuild.Pipeline {
 		}))
 	}
 
+	if len(p.CACerts) > 0 {
+		for _, cc := range p.CACerts {
+			files, err := osbuild.NewCAFileNodes(cc)
+			if err != nil {
+				panic(err.Error())
+			}
+
+			if len(files) > 0 {
+				pipeline.AddStages(osbuild.GenFileNodesStages(files)...)
+			}
```

Comment:
These files should be added to `p.Files` so that they're returned from the pipeline's inline file getter:
https://github.com/osbuild/images/blob/7f9b993f83ed0f8f4227cd5912f5be3c2f9fc7b4/pkg/manifest/os.go#L894-L903

Which is called here:
https://github.com/osbuild/images/blob/abb3328d2718431316c5c5a6674335f525b07889/pkg/manifest/manifest.go#L154-L164
so that all the needed sources are collected for the manifest generation.